src/whidbey/repl.clj

```diff
@@ -0,0 +1,11 @@
+(ns whidbey.repl
+  (:require
+    [clojure.tools.nrepl.middleware.pr-values :refer [pr-values]]
+    [whidbey.render :as render]))
+
+
+(defn init!
+  "Initializes the repl to use Whidbey's customizations."
+  [options]
+  (render/update-options! options)
+  (alter-var-root #'pr-values (constantly identity)))
```

Comment:
Ah, missed this at first glance. You can disregard my comment about front-appending, then.